config/graphql/types/Other.type.yml

```diff
@@ -108,3 +108,14 @@ IndexerStatus:
       updated: { type: "Int" }
       errors: { type: "Int" }
       statusLine: { type: "String" }
+
+Link:
+  type: "object"
+  config:
+    fields:
+      url : { type: "String!" }
+      label : { type: "String" }
+      ariaLabel : { type: "String" }
+      description: { type: "String" }
+      opensNewWindow: { type: "Boolean!" }
+      isExternal: { type: "Boolean!" }
```

Comment:
Do we need an `is` prefix here. This is now inconsistent with `opensNewWindow`, isn't it?

Reply:
Yeah I quickly noticed that the Converter namespace is weird. I simply moved the logic to `ResourceLinkResolver` to be analogous to the other resolvers, as explained here: https://github.com/sitepark/atoolo-graphql-search-bundle/pull/60#discussion_r1745289363

Regarding the `opensNewWindow`: I'm not sure, but I think `opensNewWindow` does already correctly suggest a boolean (because of the s in "opens") while `isOpensNewWindow` sounds redundant and unnatural.  `external` however would not necessarily suggest a boolean, therefore I added an "is".